tests/test_dataframe_mapper.py

```diff
@@ -54,7 +54,7 @@ def test_with_iris_dataframe(iris_dataframe):
 def test_with_car_dataframe(cars_dataframe):
     pipeline = Pipeline([
         ("preprocess", DataFrameMapper([
-            ("description", CountVectorizer()),
+            ("description", [PassthroughTransformer(), CountVectorizer()]),
```

Comment:
This fails for me:

    def transform(self, X):
    >       return np.array(X).astype(np.float)
    E       ValueError: invalid literal for float(): 1.2 16v auto corsa 3 door, Full mot with no advisories, Runs and drives great, Few age related marks but all round good condition, Good on fuel and cheap to run and insure, BLUE, 4 owners, Next MOT du

    sklearn_pandas/__init__.py:54: ValueError

This is because the `PassthroughTransformer` tries to transform the fields to float when they are not. Did you run these tests yourself??

Anyhow, I think you should create a new test method, not just augment a current one, to test exactly for this "list of transformers" feature.